x-pack/test_serverless/api_integration/test_suites/observability/config.feature_flags.ts

```diff
@@ -21,7 +21,6 @@ export default createTestConfig({
   services,
   // add feature flags
   kbnServerArgs: [
-    '--xpack.infra.enabled=true',
```

Comment:
Giving the 👍🏾 as I'm not sure why security solution is a code owner for an observability file? @crespocarlos .

Reply:
I guess we need to change this https://github.com/crespocarlos/kibana/blob/main/.github/CODEOWNERS#L1625